SUMO-252682: Adding monitors info for otel apps - set 5 (#4969)
SumoLogic/sumologic-documentation: docs/integrations/pci-compliance/opentelemetry/linux-opentelemetry.md

- Monitor actions performed by users with administrative privileges.

<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/PCI-Compliance-For-Linux/OpenTelemetry/PCI-Compliance-Req-10.png')} alt="PCI Compliance for Linux dashboards" style={{border: '1px solid gray'}}/>

> ## Create monitors for PCI Compliance for Linux - OpenTelemetry app

Reviewer: ```suggestion
## Create monitors for Linux app
```

Reviewer 2: same feedback we have multiple linux based apps

import CreateMonitors from '../../../reuse/apps/create-monitors.md';

<CreateMonitors/>

> ### PCI Compliance for Linux - OpenTelemetry Alerts

Reviewer: ```suggestion
### Linux alerts
```

Reviewer 2: same feedback we have multiple linux based apps

| Name | Description | Alert Condition | Recover Condition |
|:--|:--|:--|:--|

> | `PCI Linux - Excessive Failed Authentication` | This alert is triggered when multiple failed login attempts are detected over a 5-minute period, indicating potential brute force attempts and addressing PCI Requirement 10.2.4 for invalid logical access attempts. | Count > 5 | Count <= 5 |

Reviewer 3: ```suggestion
| `PCI Linux - Excessive Failed Authentication` | This alert is triggered when multiple failed login attempts are detected over a 5-minute period, indicating potential brute force attempts and addressing PCI Requirement `10.2.4` for invalid logical access attempts. | Count > 5 | Count <= 5 |
```

> | `PCI Linux - Privileged User Account Changes` | This alert is triggered when privileged user accounts (UID < 1000 or root accounts) are created, deleted, or modified, addressing PCI Requirement 10.2.5 for changes to identification and authentication mechanisms. | Count > 0 | Count <= 0 |

Reviewer 3: ```suggestion
| `PCI Linux - Privileged User Account Changes` | This alert is triggered when privileged user accounts (UID < 1000 or root accounts) are created, deleted, or modified, addressing PCI Requirement `10.2.5` for changes to identification and authentication mechanisms. | Count > 0 | Count <= 0 |
```

> | `PCI Linux - Unauthorized Sudo Elevation` | This alert is triggered when unauthorized users attempt to use sudo, addressing PCI Requirement 7.2 for implementing an access control system for system components with multiple users. | Count > 2 | Count <= 2 |

Reviewer: ```suggestion
| `PCI Linux - Unauthorized Sudo Elevation` | This alert is triggered when unauthorized users attempt to use sudo, which addresses PCI Requirement 7.2 for implementing an access control system among system components with multiple users. | Count > 2 | Count <= 2 |
```

> | `PCI Linux - Unauthorized Sudo Elevation` | This alert is triggered when unauthorized users attempt to use sudo, which addresses PCI Requirement 7.2 for implementing an access control system among system components with multiple users. | Count > 2 | Count <= 2 |

Reviewer 3: ```suggestion
| `PCI Linux - Unauthorized Sudo Elevation` | This alert is triggered when unauthorized users attempt to use sudo is detected, which addresses PCI Requirement `7.2.0` for implementing an access control system among system components with multiple users. | Count > 2 | Count <= 2 |
```
